token/token.go

```diff
@@ -114,6 +114,7 @@ const (
 	AND
 	LEFTSHIFT
 	RIGHTSHIFT
+	LAMBDA // => lambda short cut: `a,b => a+b`` alias for `func(a,b) {a+b}``
```

Comment:
I don't get the double backtick, maybe  you wanted this

```suggestion
	LAMBDA // => lambda short cut: `a,b => a+b` alias for `func(a,b) {a+b}`
```